### ISSUE TITLE
[bufix-2.0.x latest] DUE+RAMPS-FD V2 DEACTIVATE_SERVOS_AFTER_MOVE cause extruder auto fans not working

### ISSUE THREAD

**Issue by scott0122** [MarlinFirmware/Marlin#11561]
#11537

I'm using `SWITCHING_NOZZLE`.
When I enable `DEACTIVATE_SERVOS_AFTER_MOVE`, extruder auto fans stop working after `servo.detach()` is called.
But when I disable `DEACTIVATE_SERVOS_AFTER_MOVE`, extruder auto fans work fine.

So I enable `DEACTIVATE_SERVOS_AFTER_MOVE` and modify `src\HAL\servo.cpp`. Extruder auto fans work. Do extruder auto fans share the same timer with servos?

```cpp
void Servo::detach() {
  servo_info[this->servoIndex].Pin.isActive = false;
  /*
  timer16_Sequence_t timer = SERVO_INDEX_TO_TIMER(servoIndex);
  if (!isTimerActive(timer)) finISR(timer);
  */
}
```

**Comment by thinkyhead** [comment]
Solved?

**Comment by scott0122** [comment]
@thinkyhead 

No, still not working.

**Comment by boelle** [comment]
@scott0122 tried latest bugfix 2.0?

**Comment by boelle** [comment]
@thinkyhead i think we can close this one
